documentation/pages/move-registry/tooling/typescript-sdk.mdx

```diff
@@ -4,47 +4,93 @@ import Beta from '../../../components/beta'
 
 <Beta/>
 
-# Plugin for Sui Typescript SDK 
+# Plugins for Sui TypeScript SDK 
 
 The Move Registry (MVR, pronounced "mover") Sui TypeScript SDK plugin offers builders a seamless experience when constructing programmable transaction blocks (PTBs) using MVR names. 
 
 The plugin resolves MVR names to their respective addresses before constructing the PTB, caching the results during each runtime.
 
 This also applies to type names, especially across package upgrades.
 
+## Recommended usage
+
+The MVR plugin uses a runtime API to resolve named packages and types. While this is beneficial to the user experience, it naturally comes with considerations around performance, availability, and security.
+
+For this reason, consider using the `@mysten/mvr-static` plugin, which statically resolves all the package addresses and types during the build. The plugin caches your MVR references  
```

Comment:
@manos i tried to get rid of "pre-resolve" but you might need to change back if i messed up the meaning. One more below.